spikeinterface/core/curationsorting.py

```diff
@@ -0,0 +1,119 @@
+from .mergesorting import MergeSorting
+from .splitsortingunit import SplitSortingUnit
+import networkx as nx
+from collections import namedtuple
+node_t = namedtuple('node_t', 'id stage')
+
+class CurationSorting():
+    """
+    Class that handles curation of a Sorting object.
+
+    """
+
+    def __init__(self, parent_sorting):
+        #to allow undo and redo a list of sortingextractors is keep
+        self._sorting_stages = [parent_sorting]
+        self._sorting_stages_i = 0
+        parent_units = parent_sorting.get_unit_ids()
+        #to easily allow undo and redo a list of graphs with the history of the curation is keep
+        self._graphs = [nx.DiGraph()]
+        self._graphs[0].add_nodes_from([node_t(u,0) for u in parent_units])
+
+        self._kwargs = dict(parent_sorting=parent_sorting.to_dict())
+
+    def _get_unused_id(self):
+        #check units in the graph to the next unused unit id
+        nodes = [n.id for n in self._graphs[self._sorting_stages_i].nodes()]
+        return max(nodes)+1
+
+    def split(self,unit2split, indices_list):
```

Comment:
Should `new_unit_ids` be an optional arg as well? Note that unit ids could also be strings!

Reply:
Ok, I will check for strings as units. Probably an int cast at the begging 

The comment position is wrong. You said something like by default the split units id will be the next available units id?